plugins/mcreator-core/blockly/js/mcreator_extensions.js

```diff
@@ -205,6 +205,26 @@ Blockly.Extensions.register('replace_sphere_validator', validateIntProviderInput
 
 Blockly.Extensions.register('simple_column_validator', validateIntProviderInputs(['height', 0, Infinity]));
 
+// Helper function for extensions that validate one or more resource location text fields
+function validateResourceLocationFields(...fields) {

```

Comment:
Is it fine that I keep all "usual" mutators after extensions or that should be moved to a separate PR/commit?

Reply:
I'd prefer if you didn't move unrelated functions around, as that adds more unrelated changes to review